scipy-1.0/paper.tex

```diff
@@ -693,11 +693,11 @@ \subsection*{Project scope}
 PyMC3\cite{10.7717/peerj-cs.55}, emcee\cite{2013PASP-emcee} and 
 PyStan\cite{pystan-ref} cover Bayesian statistics and probabilistic modeling.
 scikit-image\cite{vanderwalt2014scikit} provides image processing
-capabilities beyond \texttt{scipy.ndimage}, sympy\cite{meurer2017sympy}
+capabilities beyond SciPy's \texttt{ndimage}, sympy\cite{meurer2017sympy}
 provides a Python interface for symbolic computation, and
 NetworkX\cite{hagberg2008networkx} is a Python library for probing
 graphs and networks in a more specialized way than SciPy components
-like \texttt{sparse.csgraph} or \texttt{spatial}.
+like \texttt{sparse.csgraph} or \texttt{spatial} offer.
```

Comment:
Consider the simplified sentence:
"NetworkX is a Python library for probing graphs in a more specialized way than sparse.csgraph."
vs 
"NetworkX is a Python library for probing graphs in a more specialized way than sparse.csgraph offers."
I'm sure that the first version can be improved, but I'm not convinced that appending "offers" is the way to do it.

Reply:
Ambivalent